packages/liquidator/src/liquidator.js

```diff
@@ -232,11 +268,14 @@ class Liquidator {
           position: position,
           minLiquidationPrice: this.liquidationMinPrice,
           maxLiquidationPrice: maxCollateralPerToken.toString(),
-          tokensToLiquidate: tokensToLiquidate.toString(),
+          tokensToLiquidate: "0",
+          syntheticTokenBalance: syntheticTokenBalance.toString(),
           error: new Error("Refusing to liquidate 0 tokens")
         });
         continue;
       }
+      const [, , , { rawValue: tokensToLiquidateString }] = liquidationArgs;
```

Comment:
What are these ","'s? They look weird

Reply:
it returns an array of arguments, its just a way to destructure and ignore the positions you dont care about, rather than trying to assign them to a variable.

Reply:
would it be better to return these values an object of named values?

Reply:
it would be messier to return an object, as it would need to be restructured to be passed in directly to the function call:
`this.empContract.methods.createLiquidation(...liquidationArgs);` 

This is really only needed because of the logs below. I can find a way to make it cleaner.

Reply:
I know it doesn't use destructuring syntax, but would this be easier to read?
```
const tokensToLiquidateString = liquidationArgs[3].rawValue;
```